apps/web/src/pages/templates/components/InputVariables.tsx

```diff
@@ -18,11 +18,11 @@ export const InputVariables = ({
   }
 
   try {
-    const module = require('@novu/ee-billing-web');
+    const module = require('@novu/ee-echo-web');
     const InputVariablesComponent = module.InputVariables;
 
     return <InputVariablesComponent schema={inputs} onSubmit={onSubmit} onChange={onChange} />;
-  } catch (e) {}
-
-  return null;
+  } catch (e) {
+    throw e;
+  }
```

Comment:
I'm curious if there's a rationale behind returning null when it's not self-hosted. In my opinion, we should consider the web failed if the import was intended but unsuccessful.

Reply:
Valid point. I also tend to agree with this

Reply:
Thanks for the feedback, awesome so i will keep the throw here